Document submodule install when installing Kuzzle manually (#234)
kuzzleio/documentation: src/guide/1/getting-started/first-steps/index.md

If you are performing a clean install you might see some `UNMET PEER DEPENDENCY` warnings, these are safe to ignore as they refer to optional dependencies.
</div>

Then, create an `init.js` file and start by loading the Kuzzle Javascript SDK.

> Next we will instantiate a client that will automatically connect to Kuzzle via websockets. Replace 'kuzzle' with the corresponding server name or ip address, or localhost :

Author: Tone consistency:

```suggestion
Next, instantiate a client that automatically connects to Kuzzle via WebSocket. Replace `'kuzzle'` with the corresponding server name or IP address:
```

node create.js
```

> Your console should output the following message:

Author: ```suggestion
Your console should show the following message:
```

Kuzzle provides pub/sub features that can be used to trigger real-time notifications based on the state of your data (for a deep-dive on notifications check out the **Room** class definition in the <a href="{{ site_base_path }}sdk-reference">SDK Reference</a>).

> Let's get started. Complete your `create.js` file :

Author: ```suggestion
Let's get started. Complete your `create.js` file:
```

  [snippet=subscribe]

Run your file in Node.js

```bash
node create.js
```

> This will create a new document in Kuzzle which will trigger a [notification]({{ site_base_path }}guide/1/essentials/real-time) :

Author: ```suggestion
This creates a new document in Kuzzle, triggering a [notification]({{ site_base_path }}guide/1/essentials/real-time):
```
